source/mge-pickle.js

```diff
@@ -0,0 +1,563 @@
+/* jshint esversion: 6 */
+
+var pickle = pickle || {};
+
+pickle.Unpickler = class {
```

Comment:
Remove duplicate pickle reader and use the one in `pickle.js`.

Reply:
Line 239 in pickle.js don't support the Megengine files, so I have to copy a old version.
```
 else if (ArrayBuffer.isView(state) || Object(state) !== state) {
                        throw new pickle.Error('Invalid state dict' + (obj && obj.__module__ && obj.__name__ ? " for '" + obj.__module__ + '.' + obj.__name__ + "'": '') + '.');
                    }
```

Reply:
Can you share a repro and fix `pickle.js` to work for all scenarios.